tools/install/installer/installer.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/spf13/pflag"
+	"os"
+	"path/filepath"
+
+	"code-intelligence.com/cifuzz/pkg/install"
+	"code-intelligence.com/cifuzz/pkg/log"
+	"code-intelligence.com/cifuzz/tools/install/bundler/embed"
+)
+
+func main() {
+	flags := pflag.NewFlagSet("cifuzz installer", pflag.ExitOnError)
+	installDir := flags.StringP("install-dir", "i", "~/cifuzz", "The directory to install cifuzz in")
+	helpRequested := flags.BoolP("help", "h", false, "")
+
+	if err := flags.Parse(os.Args); err != nil {
+		log.Error(err, err.Error())
+		os.Exit(1)
+	}
+
+	if *helpRequested {
+		log.Printf("Usage of %s:\n", "cifuzz installer")
+		flags.PrintDefaults()
+		os.Exit(0)
+	}
+
+	fs := &embed.Bundle
+	err := install.ExtractBundle(*installDir, fs)
+	if err != nil {
```

Comment:
```suggestion
	if err := install.ExtractBundle(*installDir, fs); err != nil {
```
Would be consistent with the error handling above in this file